numpy/core/tests/test_regression.py

```diff
@@ -45,11 +45,6 @@ def test_pickle_transposed(self):
         f.close()
         assert_array_equal(a, b)
 
-    def test_typeNA(self):
-        # Ticket #31
-        assert_equal(np.typeNA[np.int64], 'Int64')
-        assert_equal(np.typeNA[np.uint64], 'UInt64')
```

Comment:
Better to mask the deprecations and keep the test here.

Reply:
adopted